common/governments/civics/hfe_civic.txt

```diff
@@ -111,4 +106,80 @@ hfe_civic_auto_paradise = {
 		# army_experience_gain_mult = 1
 		# general_skill_levels = 2
 	# }
-# }
\ No newline at end of file
+# }
+
+#############################################################
+#	END OF CIVICS	###
+##############################
+#	BEGIN ORIGINS	###
+###################################
+
+hfe_civic_egypt_rulers_origin = {
+#	modification = no
+	is_origin = yes
+#	pickable_at_start = no
+	icon = "gfx/event_pictures/origins/ju_ahet_origin_icon.dds"
+	picture = "GFX_HFE_TSP_ju_ahet_origin"
+
+	playable = { always = no }
+
+#	potential = {
+#		always = 
+#		country_type = {
+#			value = human_fallen_empire
+#		}
+#	}
+
+	possible = {
+	}
+
+	random_weight = {
+		base = -1
+		modifier = {
```

Comment:
this node is not needed
Always -1 will prevent it from random picking.

Reply:
Okay, done